ansible/roles/ci/doc/setup/shippable.md

```diff
@@ -0,0 +1,26 @@
+# Shippable Server
+
+## How to setup
+
+The following repository can be used as an example of [Shippable](https://wwww.shippable.com/) configuration [build servers check](https://github.com/shadow-robot/build-servers-check)
+
+The process of setting up [Shippable](https://wwww.shippable.com/) server
+
+  * In the root folder of your repository
+    * Copy *shippable.yml* from [build servers check](https://github.com/shadow-robot/build-servers-check) repository. 
+    * Remove *env* section and *after_failure* sections.
+    * Create *repository.rosinstall* in case if you want to install some packages from source code. This file should be in in [rosinstall](http://wiki.ros.org/rosinstall) format.
```

Comment:
Create *repository.rosinstall* in case you want to install some packages from source code. This file should be in [rosinstall](http://wiki.ros.org/rosinstall) format.